src/viking.js

```diff
@@ -1,11 +1,53 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength){
+    this.health = health;
+    this.strength = strength;
+  }
+  attack(){
+    return this.strength;
+  }
+  receiveDamage(damage){
+    return this.health -= damage
+  }  
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength){
+    super(health, strength)
+    this.name = name
+  }
+  attack(){
+    return this.strength;
+  }
+  receiveDamage(damage){
+    this.health -= damage;
+    if(this.health > 0){
+      return `${this.name} has recived ${damage} points of damage` 
+    } else {
+      `${this.name} has died in act of combat"`
+    }
```

Comment:
You must have keyword `return`  in each if else statement. 

If the first condition is falsy, we go into else -> and it is missing `return` now